src/utils/calendar_manager.ts

```diff
@@ -209,6 +211,8 @@ const handleParticipants = async (
     participants.filter(p => p.account_address).map(p => p.account_address!)
   )
 
+  const sharedSlotId = uuidv4()
```

Comment:
All participants must have unique slotIds including guests. This would prevent account users from scheduling a meeting.